yarn.lock

```diff
@@ -3021,6 +3291,22 @@
     "@radix-ui/react-use-size" "1.0.1"
     "@radix-ui/rect" "1.0.1"
 
+"@radix-ui/react-popper@1.2.0":
```

Comment:
It's not ideal to have two versions of the same library. It looks like we're already importing several `@radix-ui` libraries via Storybook. Should we look at upgrading to get the versions in sync?